zebra-chain/src/chain_tip.rs

```diff
@@ -58,6 +58,21 @@ pub trait ChainTip {
 
         Some(estimator.estimate_height_at(now))
     }
+
+    /// Return an estimate of how many blocks there are ahead of zebra's best chain tip
+    /// until the network chain tip.
+    ///
+    /// The estimate is calculated based on the current local time, the block time of the best tip
+    /// and the height of the best tip.
+    ///
+    /// This estimate may be negative if the current local time is behind the chain tip block's timestamp.
```

Comment:
Nice

Reply:
Do we know why past times are handled this way?

https://github.com/ZcashFoundation/zebra/blob/main/zebra-chain/src/chain_tip/network_chain_tip_height_estimator.rs#L122-L137

Reply:
I don't recall, but I'd be willing to put money on "this is how zcashd handles it"